Gopkg.toml

```diff
@@ -123,5 +120,6 @@ ignored = [
   version = "kubernetes-1.14.0"
 
 [[override]]
-  name = "k8s.io/client-go"
-  version = "v11.0.0"
+  name = "k8s.io/kube-openapi"
+  #revision matches apimachinery and apiserver
+  revision = "b3a7cee44a305be0a69e1b9ac03018307287e1b0"
```

Comment:
can this one not be a semver like the other k8s.io dependencies?

Reply:
fixed with revert